### DISCUSSION TITLE
Using M4 button as the "left-click" function of a mouse

### DISCUSSION THREAD

**Discussion by jego5864** [seerge/g-helper#699] [Q&A]
Hi all,

I have been on the hunt for a way to re-map the M4 key to serve as the left-click function of the mouse, and I've come up empty. I'm trying to emulate a feature I miss from my previous laptop (a Lenovo ThinkPad), which has the three buttons located at the top of the touchpad. Repurposing an unused key to provide a similar function would be a godsend.

Any suggestions for how to go about doing this? I've already looked at Windows PowerToys, SharpKeys, and KeyTweak to make this happen, but it'd be amazing if I could write this into Ghelper somehow.

Thanks!

**Comment by seerge** [comment]
@jego5864 hello, you can do that by setting a custom key action as explained in FAQ https://github.com/seerge/g-helper#custom-hotkey-actions

`` VK_LBUTTON | 0x01 | Left mouse button`` 
https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes

**Reply by jego5864** [reply]
Thanks for the reply! I tried that yesterday and was unable to get it to work. I just downloaded the pre-release of v92 and tried again, and it still doesn't work. I _was_ able to custom re-map M4 to the letter "a" when I was first toying around with this, so I wonder if it's the left mouse button key code itself that is incompatible with this operation for some reason.

**Reply by seerge** [reply]
Alternatively you can remap m4 to f13-24 key (that doesn’t exist on keyboard), and then in turn bind this f13 to whatever action you want using autohotkey

**Reply by jego5864** [reply]
That is a great idea. Thank you! I will give that a try.

**Reply by Nic0laus** [reply]
I have the same case, where I'm unable to get any mouse button to work, while for example letters do actually work. Do you know if there is any particular reason why it doesn't work? (0.129.0)

**Reply by seerge** [reply]
Same advice - to try autohotkey :)
